src/python/grpcio/support.py

```diff
@@ -44,8 +44,10 @@
 int main(int argc, char **argv) { return 0; }
 """
 C_PYTHON_DEV_ERROR_MESSAGE = """
-Could not find <Python.h>. This could mean the following:
-  * You're on Ubuntu and haven't `apt-get install`ed `python-dev`.
+Could not find <python.h>. This could mean the following:
+  * You're on Ubuntu and haven't `apt-get install python-dev`.
```

Comment:
Given that you're changing the part-of-speech of the command, mind fixing the `haven't` to `haven't run`?